server/README.md

```diff
@@ -168,15 +166,15 @@ We use [helm](https://helm.sh/docs/) to manage our deployments. Helm allows us t
 
 1. on your local machine, open Visual Studio Code in the `firmware/boron-ins-fsm` directory and open the Particle Workbench extension
```

Comment:
I just realized that all of the ministeps start with 1. instead of in increasing number order. I'm not sure if there's a quick way to fix this though!

Reply:
This is done on purpose. In Markdown, using `1.` will auto-increment the ordered list. It makes it easier for the future in case you need to add a new step in the middle somewhere then you don't need to manually update all the numbers.